swarm/storage/localstore.go

```diff
@@ -28,8 +28,9 @@ import (
 
 type LocalStoreParams struct {
 	*StoreParams
-	ChunkDbPath string
-	Validators  []ChunkValidator `toml:"-"`
+	ChunkDbPath    string
+	Validators     []ChunkValidator `toml:"-"`
+	SkipValidation bool
```

Comment:
If you have access to this struct and can set the `SkipValidation`, why can't we just set the `Validators` to an empty slice?